eslint-config-base/index.js

```diff
@@ -190,5 +378,27 @@ module.exports = {
     'import/no-self-import': 1,
     'import/no-cycle': 0,
     'import/no-relative-parent-imports': 0,
+    'import/prefer-default-export': 0,
+    'import/no-extraneous-dependencies': [2, {
+      devDependencies: [
+        'test/**', // tape, common npm pattern
+        'tests/**', // also common npm pattern
+        'spec/**', // mocha, rspec-like pattern
+        '**/__tests__/**', // jest pattern
+        '**/__mocks__/**', // jest pattern
+        'test.{js,jsx}', // repos with a single test file
+        'test-*.{js,jsx}', // repos with multiple top-level test files
+        '**/*{.,_}{test,spec}.{js,jsx}', // tests where the extension or filename suffix denotes that it is a test
```

Comment:
Looks like we should add `ts/tsx?` extensions also.
```suggestion
        'test.{js,jsx,ts,tsx}', // repos with a single test file
        'test-*.{js,jsx,ts,tsx}', // repos with multiple top-level test files
        '**/*{.,_}{test,spec}.{js,jsx,ts,tsx}', // tests where the extension or filename suffix denotes that it is a test
```